### PR TITLE
Adding Results for TCGA Classify Module

### PR DESCRIPTION
**Pull request by gwaygenomics**
Here I add the results and figures for the TCGA classify module

## Main Figure

tcga_biobombe_main_figure

## Supplementary Figure

supplementary_figure_tcga_classify_auroc

### CODE REVIEW

File: 9.tcga-classify/scripts/viz_util.R [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+#
+# Usage:
+# source("scripts/viz_util.R")
+
+load_results <- function(results_path, file_string = "classify_metrics",
+                         process_output = FALSE) {
+  # From the finalized results, load and process specific result files
+  #
+  # Arguments:
+  # results_path - the file path to scan for specific results
```
Comment (ajlee21) [inline 9.tcga-classify/scripts/viz_util.R]: What type of results is it expecting?

File: 9.tcga-classify/scripts/viz_util.R [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+  # file_string - a string indicating which type of data to load in
```
Comment (ajlee21) [inline 9.tcga-classify/scripts/viz_util.R]: What possible string values are there?

File: 9.tcga-classify/scripts/viz_util.R [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+  # process_output - boolean indicating if the output should be processed
```
Comment (ajlee21) [inline 9.tcga-classify/scripts/viz_util.R]: Otherwise?

File: 9.tcga-classify/scripts/viz_util.R [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+  }
+
+  return(return_obj)
+}
+
+process_results <- function(df, raw_df, data_type = "cv") {
+  # Filter and arrange metrics for plotting input
+  #
+  # Arguments:
+  # df - a dataframe storing classification results
```
Comment (ajlee21) [inline 9.tcga-classify/scripts/viz_util.R]: What is the format of this dataframe?  row? col?  values?

**Comment by gwaygenomics** [comment]
Amazing comments - will address in next commit!

**Pull request merged**